apps/codeforafrica/src/components/Opportunities/Opportunities.js

```diff
@@ -0,0 +1,100 @@
+import { Section } from "@commons-ui/core";
+import { styled } from "@mui/material/styles";
+import React, { useEffect, useMemo, useRef, useState } from "react";
+
+import ChoiceChip from "@/codeforafrica/components/ChoiceChip";
+import ChoiceChipGroup from "@/codeforafrica/components/ChoiceChipGroup";
+import NextPreviousPagination from "@/codeforafrica/components/NextPreviousPagination";
+import OpportunityCardList from "@/codeforafrica/components/OpportunityCardList";
+
+const ALL_TAGS = "All";
+
+const OpportuniesRoot = styled("div")({
+  scrollMarginTop: 32,
+});
+
+const computePagination = (all, page, pageSize) => {
+  const count = Math.ceil(all.length / pageSize);
+  const opportunities = all.slice((page - 1) * pageSize, page * pageSize);
+  return { count, opportunities };
+};
+
+function Opportunies(props) {
+  const { opportunities = [], page: pageProp = 1, pageSize = 4 } = props;
+  const ref = useRef();
+  const [tags] = useState(() => {
+    return [ALL_TAGS, ...new Set(opportunities?.flatMap((a) => a.tags || []))];
+  });
+  const [selectedTag, setSelectedTag] = useState(ALL_TAGS);
+  const [page, setPage] = useState(pageProp);
+  const [pagination, setPagination] = useState(() => {
+    return computePagination(opportunities, page, pageSize);
+  });
+  const filteredOpportunies = useMemo(() => {
+    if (selectedTag === ALL_TAGS) {
+      return opportunities;
+    }
+    return opportunities.filter((p) =>
```

Comment:
This works @kilemensi  but shouldn't we support multiple tags filtering? Right now I can't find Opportunities in Kenya and Remote.

Reply:
Possible @esirK and may be we will:

1. Looking at the design and looking an [Material UI](https://material.io/components/chips#choice-chips), these chips look more like Choice chips than Filter chips.
2. This exact same approach is taken for Projects, Stories, Team members and Opportunites. Once we know if all or some need to support Filtering while others Choosing, we can refactor.
3. There is also a search field that is currently missing (need CMS/API). Choice + search should be able to do Kenya + Remote.